fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/digitalcash/PostTransaction.java

```diff
@@ -0,0 +1,56 @@
+package com.github.dedis.popstellar.model.network.method.message.data.digitalcash;
+
+import com.github.dedis.popstellar.model.network.method.message.data.Action;
+import com.github.dedis.popstellar.model.network.method.message.data.Data;
+import com.github.dedis.popstellar.model.network.method.message.data.Objects;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Base64;
+
+public class PostTransaction extends Data {
+  @SerializedName(value = "transaction_id")
+  private final String transaction_id; // TxOutHash SHA256 over base64encode(transaction)
+
+  @SerializedName(value = "transaction")
+  private final Transaction transaction; // the transaction object // String
+
+  /**
+   * @param transaction the transaction object
+   */
+  public PostTransaction(Transaction transaction) {
+    this.transaction = transaction;
+    this.transaction_id = Base64.getEncoder().encodeToString(transaction.toString().getBytes());
```

Comment:
I have a question here 
How do we encode a transaction in base64 ?  
And then how do we hash it in sha256 and get a String at the end ?